reference/glossary.md

```diff
@@ -0,0 +1,445 @@
+- [ADB](#adb)
+- [AMC](#amc)
+- [AMS](#ams)
+- [AMS Node Controller](#ams-node-controller)
+- [AMS SDK](#ams-sdk)
+- [APK](#apk)
+- [AWS](#aws)
+- [Addon](#addon)
+- [Amazon Web Services](#aws)
+- [Anbox](#anbox)
+- [Anbox Application Manager](#aam)
+- [Anbox Application Registry](#aar)
+- [Anbox Cloud](#anbox-cloud)
+- [Anbox Cloud Appliance](#anbox-cloud-appliance)
+- [Anbox Management Client](#amc)
+- [Anbox Management Service](#ams)
+- [Anbox Platform SDK](#anbox-platform-sdk)
+- [Anbox Streaming SDK](#anbox-streaming-sdk)
+- [Android Archive](#android-archive)
+- [Android Debug Bridge](#adb)
+- [Android Package Kit](#apk)
+- [Android app](#android-app)
+- [Appium](#appium)
+- [Application](#application)
+- [Application container](#application-container)
+- [Application manifest](#application-manifest)
+- [Base container](#base-container)
+- [Boot package](#boot-package)
+- [Bootstrap process](#bootstrap-process)
+- [Container](#container)
+- [Core stack](#core-stack)
+- [Coturn](#coturn)
+- [GPU](#gpu)
+- [Grafana](#grafana)
+- [Graphics Processing Unit](#gpu)
+- [HA](#ha)
+- [High availability](#ha)
+- [Hook](#hook)
+- [Image](#image)
+- [Instance type](#instance-type)
+- [Juju](#juju)
+- [LXD](#lxd)
+- [Manifest](#application-manifest)
+- [Monitoring stack](#monitoring-stack)
+- [NATS](#nats)
+- [Nagios](#nagios)
+- [Neural Autonomic Transport System](#nats)
+- [Node controller](#ams-node-controller)
+- [Platform](#platform)
+- [Prometheus](#prometheus)
+- [Raw container](#raw-container)
+- [Regular container](#regular-container)
+- [STUN/TURN server](#stun/turn-server)
+- [Scrcpy](#scrcpy)
+- [Session](#session)
+- [Snap](#snap)
+- [Software Rasterization](#swrast)
+- [Stream agent](#stream-agent)
+- [Stream gateway](#stream-gateway)
+- [Streaming stack](#streaming-stack)
+- [Swrast](#swrast)
+- [Ubuntu Advantage for Applications](#ubuntu-advantage-for-applications)
+- [Ubuntu One](#ubuntu-one)
+- [Watchdog](#watchdog)
+- [Web dashboard](#web-dashboard)
+- [WebRTC](#webrtc)
+
+[/Details]
+
+## Definitions
+
+<a name="addon"></a>
+### Addon
+
+A piece of code that can be used to extend and customise images in Anbox Cloud.
+
+See https://discourse.ubuntu.com/t/addons/25293.
+
+<a name="aws"></a>
+### Amazon Web Services (AWS)
+
+A cloud platform provided by Amazon that can be used to host Anbox Cloud.
+
+See [the AWS website](https://aws.amazon.com/).
+
+<a name="ams-node-controller"></a>
+### AMS Node Controller
+
+A service that runs on every LXD node and puts the appropriate firewall rules in place when a container is started or stopped.
+
+<a name="ams-sdk"></a>
+### AMS SDK
+
+An SDK that provides Go code for connecting to AMS through the exposed REST API.
+
+See [AMS SDK](https://discourse.ubuntu.com/t/anbox-cloud-sdks/17844#ams-sdk).
+
+<a name="anbox"></a>
+### Anbox
+
+An open-source project that provides a container-based approach to boot a full Android system on a regular GNU/Linux system. Anbox Cloud uses Anbox.
```

Comment:
Can we add more information on how Anbox Cloud uses Anbox? I didn't find anything.

Reply:
Do we want to specify this here ? The open source Anbox and Anbox Cloud are starting to diverge in quite a few ways now

Reply:
I think it's important to point out that there is something called Anbox and that it is different from Anbox Cloud, and to do that, we need to explain how they differ.

Reply:
Having an explanation here is great as this is a thing which is often misunderstood (and the naming doesn't make it any better). Reality is that Anbox and Anbox Cloud are completely independent things. They don't share nothing but the name "Anbox" and similar ideas. Technically Anbox Cloud is not using Anbox. We need to make this clear in here.

Reply:
Change to something like:
"A component of Anbox Cloud that facilitates booting an Android system on a regular GNU/Linux system. While the concepts of the Anbox component in Anbox Cloud are similar to the [Anbox open source project](https://github.com/anbox/anbox), the Anbox open source project is an independent project that is not related to or used in Anbox Cloud.